library/globals.inc.php

```diff
@@ -86,15 +86,27 @@
 // xl('Appearance')
 // xl('Locale')
 // xl('Features')
+// xl('Report')
```

Comment:
These are new lines 89-103... they should have already been there.  Makes we worried that something else in a previous commit was deleted ...